Gemfile

```diff
@@ -4,7 +4,7 @@ gem 'resque', '~>1.19'
 
 group :test do
   gem 'mocha', '~>0.9'
-  gem 'shoulda', '~>2.10.2'
+  gem 'minitest'
```

Comment:
Can you change this to an explicit require of ~>5.5? Minitest < 5 has a different API